src/ContextMenuTrigger.js

```diff
@@ -51,11 +51,10 @@ export default class ContextMenuTrigger extends Component {
         if (this.props.holdToDisplay >= 0) {
             event.persist();
 
-            this.touchHandled = true;
-
             this.touchstartTimeoutId = setTimeout(
                 () => this.handleContextClick(event),
-                this.props.holdToDisplay
+                this.props.holdToDisplay,
+                this.touchHandled = true
```

Comment:
But setTimeout takes two args, a callback and then a timeout (this.props.holdToDisplay). The way you fixed it now you left the timeout in the middle of the callback, so that's not gonna work :)